sysroot/usr/include/kernel/sys/tasks.hpp

```diff
@@ -12,13 +12,14 @@
 #ifndef PANIX_TASKS_HPP
 #define PANIX_TASKS_HPP
```

Comment:
Why not use `#pragma once`?

Reply:
See #174 and its associated MR. Resolved in 67712a8.